src/main/java/org/plos/repo/rest/RootController.java

```diff
@@ -52,7 +52,7 @@
   public Response index() {
     // TODO: display spirit animal (squirrel, foraging woodpecker ?)
```

Comment:
what?

Reply:
lol, deleted. I assume that a spirit animal would be like rhino or wombat or dipro or lemur